knexfile.js

```diff
@@ -1,5 +1,6 @@
 const NODE_ENV = process.env.NODE_ENV || 'development'
 const isProduction = NODE_ENV === 'production'
+const POSTGRES_PORT = parseInt(process.env.POSTGRES_PORT, 10) || undefined
```

Comment:
@JamesMGreene : shouldn't this be `5443`?

Reply:
@andyfeller If you're referring to the `10`, that's a confusing antiquated pattern in the [JS `parseInt` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) where the second parameter (radix) is the intended base (i.e. `10` = "parse this as a base 10 decimal number"). It's not strictly required anymore but some linters still get angry if you leave it off. 🙄 

If you're referring to the `undefined`, that is intentional to ensure that local development outside of Codespaces can still use the default PostgreSQL port (`5442`) for a local database instance.